### ISSUE TITLE
Command 'projects.editConfig' not found

### ISSUE THREAD

**Issue by Septh** [fabiospampinato/vscode-projects-plus#51]
The exact same issue as #10 (and probably #19) resurfaced today using the latest Insiders build of VS Code.

```
Version: 1.39.0-insider (system setup)
Commit: b0d714d2de2886060e0904bf0cc774e0d70667cd
Date: 2019-09-26T09:07:24.281Z
Electron: 6.0.9
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18362
```

This time however, the Electron console shows the following:

```
[Extension Host] debugger listening on port 9549
abstractExtensionService.ts:396 Activating extension 'fabiospampinato.vscode-projects-plus' failed: 'isExtensible' on proxy: trap result does not reflect extensibility of proxy target (which is 'true').
_logMessageInConsole @ abstractExtensionService.ts:396
console.ts:137 [Extension Host] Activating extension `fabiospampinato.vscode-projects-plus` failed:  'isExtensible' on proxy: trap result does not reflect extensibility of proxy target (which is 'true') (at e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:496:862)
t.log @ console.ts:137
console.ts:137 [Extension Host] Here is the error stack:  TypeError: 'isExtensible' on proxy: trap result does not reflect extensibility of proxy target (which is 'true')
	at Object.getExtension (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:9:86883)
	at Object._initItem (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:9:104430)
	at new <anonymous> (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:9:104163)
	at Object.<anonymous> (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:9:104124)
	at n (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:1:186)
	at Object.<anonymous> (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:9:130040)
	at n (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:1:186)
	at c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:1:985
	at Object.<anonymous> (c:\Users\Stephan\.vscode-insiders\extensions\fabiospampinato.vscode-projects-plus-1.23.0\out\extension.js:1:995)
	at Module._compile (e:\Microsoft VS Code Insiders\resources\app\out\vs\loader.js:15:749)
	at Object..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.<anonymous> (internal/modules/cjs/loader.js:560:12)
	at Function.<anonymous> (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:783:720)
	at Function.<anonymous> (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:752:302)
	at Function._load (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:748:870)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Function.s [as __$__nodeRequire] (e:\Microsoft VS Code Insiders\resources\app\out\vs\loader.js:32:963)
	at Object._loadCommonJSModule (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:753:459)
	at Object._doActivateExtension (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:664:275)
	at Object._activateExtension (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:663:360)
	at Object.actualActivateExtension (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:661:296)
	at Object._activateExtension (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:496:731)
	at e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:496:409
	at Array.map (<anonymous>)
	at Object._activateExtensions (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:496:397)
	at Object.activateByEvent (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:494:126)
	at Object._activateByEvent (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:662:24)
	at Object._handleEagerExtensions (e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:665:762)
	at e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:669:215
	at processTicksAndRejections (internal/process/task_queues.js:89:5) (at e:\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:496:933)
notificationsAlerts.ts:38 Error: command 'projects.editConfig' not found
    at Proxy._tryExecuteCommand (commandService.ts:80)
    at commandService.ts:73
    at processTicksAndRejections (internal/process/task_queues.js:89)
    at async t.ActionRunner.run (actions.ts:208)
```

The only way around is to set the `Config path` setting to a valid json file.

**Comment by fabiospampinato** [comment]
It sounds like the same issue as https://github.com/fabiospampinato/vscode-terminals/issues/32, I don't have the time to deal with them right now, but at maximum once the next insider becomes the new stable release these issues we'll get fixed.

**Comment by LeanderD** [comment]
FYI, after todays update it started working again for me.

**Comment by fabiospampinato** [comment]
I'm closing this since it appears to be working again.

**Comment by RobinD42** [comment]
This issue is now manifesting in the current stable release.

**Comment by lootboxer** [comment]
@fabiospampinato, **Will you fix this???**

**Comment by fabiospampinato** [comment]
It looks like it's about time to address it, I'll try to push a fix soon-ish 👍

**Comment by wiecznyPoszukiwacz** [comment]
Since extension stopped working, I can see how I lack it ;) Great job!

**Comment by Esteban-Rocha** [comment]
Hey @fabiospampinato, thanks for the ext. is a life saver.

That being said it is broken, none of the commands works, thus rendering the ext absolutely broken. I wish I could do a PR but VsCode `ext` coding is not something I'm familiar with so I can just add in and report that this bug seems to be hitting a large user base.

**Comment by Septh** [comment]
@wiecznyPoszukiwacz @Esteban-Rocha As said above, just set the `Config Path` setting in VS Code to circumvent this problem.

![Capture](https://user-images.githubusercontent.com/11949623/68719906-ca50de80-05ad-11ea-8711-d34075949c3c.PNG)

**Comment by Esteban-Rocha** [comment]
@Septh Thanks! totally missed that 👍

**Comment by wiecznyPoszukiwacz** [comment]
@Septh Thanks a lot! :)

**Comment by fabiospampinato** [comment]
Fixed in 1.23.1, sorry for the delay but I recently relocated to London and trying to find some more permanent accommodation can be time consuming and not upgrading or downgrading was a viable option.
